src/azure-cli/azure/cli/command_modules/storage/operations/fileshare.py

```diff
@@ -93,3 +93,57 @@ def get_share_stats(client, timeout=None, **kwargs):
 def set_share_metadata(client, metadata=None, timeout=None, **kwargs):
     client.set_share_metadata(metadata=metadata, timeout=timeout, **kwargs)
     return True
+
+
+def list_handle(client, marker, num_results, **kwargs):
+    from ..track2_util import list_generator
+    directory_path = kwargs.pop("directory_name", None)
+    if directory_path and directory_path.startswith('./'):
+        directory_path = directory_path.replace('./', '', 1)
```

Comment:
Do we really have such cases to pass in `./dir/file` as `--path`? What if customers provide `/dir/file`, should we escape the first `/`?